model/gen-ai/events.yaml

```diff
@@ -313,3 +313,88 @@ groups:
                     arguments field.
                   examples: ['{\"location\": \"Paris\"}']
                   requirement_level: opt_in
+
+  - id: event.gen_ai.evaluation.result
+    name: gen_ai.evaluation.result
+    type: event
+    stability: development
+    brief: >
+      This event describes a generic GenAI response evaluation result.
+    extends: gen_ai.common.event.attributes
+    attributes:
+      - id: gen_ai.evaluation.name
```

Comment:
you can  only declare attributes inside attribute groups, not inside signal definitions. Please run `make check-policies` locally (in WSL if you use windows).

once defined in the registry, it's possible to reference it here like other attributes